presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -811,6 +811,14 @@ Optimizer Properties
 
     Track history based plan statistics from complete plan fragments in failed queries.
 
+``optimizer.log-plans-used-in-hbo``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``false``
+
+    Log stats equivalent plan and canonicalized plans used in history based optimization.
```

Comment:
```suggestion
    Log the stats equivalent plan and canonicalized plans used in history based optimization.
```
"Log stats equivalent plan" seems a little hard to parse. Suggestion of "the" taken from the Description of the PR makes it easier to understand.